bigquery_etl/docs/mozfun/generate_mozfun_docs.py

```diff
@@ -88,7 +88,134 @@ def generate_mozfun_docs(out_dir, project_dir):
                             dataset_doc_file.write(f"{formated}\n\n")
                         # Inject the contents of the README.md
                         dataset_doc_file.write(docfile_content)
-                        # Add links to source and edit
+                        if is_udf:
+                            found_input = False
+                            input_lines = []
+                            found_return = False
+                            return_lines = []
+                            with open(os.path.join(root, UDF_FILE), "r") as udf_sql:
+                                udf_content = [
+                                    line
+                                    for line in udf_sql.readlines()
+                                    if not line.startswith("#")
+                                ]
+
+                                FUNCTION_HEADER = (
```

Comment:
Shall we use the [funtion to extract input parameters](https://github.com/mozilla/bigquery-etl/blob/3756_function_to_get_UDF_input_output/bigquery_etl/util/udf_functions.py) using regex? The [test scenarios](https://github.com/mozilla/bigquery-etl/blob/3756_function_to_get_UDF_input_output/tests/util/test_udf_functions.py) make it more robust and easy to read the expected headers. If a developer requires a different header then the function can be expanded.